src/components/site/PageHeader.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const PageHeader = props => (
+  <div
+    className="dg_page-header"
+    style={{ backgroundImage: `url(${props.backGroundImage})` }}
```

Comment:
@danfox01 wants this to be include din the markup, so instead of the css controlling this, it has to be on the page. The reason for this, is because the web services team will be managing this...not developrs. I think everything else is okay.

Reply:
done